crates/astria-conductor-test/kubernetes/test-environment.yml

```diff
@@ -1,32 +1,53 @@
 apiVersion: v1
 data:
+  generate-token.sh: |
+    #!/bin/sh -x
+    celestia bridge auth admin \
+      --node.store "$home_dir/bridge" \
+      --keyring.accname validator > "$home_dir"/.admin_token
   init-bridge.sh: |
-    #!/bin/sh
+    #!/bin/sh -x
 
     set -o errexit -o nounset
 
-    ./celestia bridge init \
+    celestia bridge init \
       --node.store "$home_dir/bridge" \
       --core.ip 127.0.0.1
     cp -r "$home_dir/keyring-test" "$home_dir/bridge/keys/"
-  start-bridge.sh: "#!/bin/bash\n\nset -o errexit -o nounset -o pipefail\n\ngenesis_hash=$(curl
-    -s -S -X GET \"http://127.0.0.1:26657/block?height=1\" | jq -r '.result.block_id.hash')\nif
-    [ -z \"$genesis_hash\" ] \nthen\n  echo \"did not receive genesis hash from celestia;
-    exiting\"\n  exit 1\nelse\n  echo \"genesis hash received: $genesis_hash\"\nfi\n\nexport
-    CELESTIA_CUSTOM=\"test:$genesis_hash\"\n  # --p2p.network \"test:$celestia_custom\"\nexport
-    GOLOG_LOG_LEVEL=\"debug\"\nexec ./celestia bridge start \\\n  --node.store \"$home_dir/bridge\"
-    \\\n  --gateway \\\n  --keyring.accname \"$validator_key_name\"\n"
+  start-bridge.sh: |
```

Comment:
more of a kubernetes question but what does this file do? why are the start-bridge.sh script contents copied here?

Reply:
@noot this is a k8s way to avoid having to mount files from the host into the container (the way you'd do in podman or docker). Instead the scripts can be inlined into config maps, and then these can be mounted into the containers using a `configMap:` entry under `volumes:`

You can read more here: https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#add-configmap-data-to-a-volume